README.md

```diff
@@ -89,31 +89,32 @@ USAGE: inspec_tools generate_inspec_metadata
 
 ## Docker Usage
 
-Building the Docker container and running it with `docker run -v`pwd`:/share -e CHEF_LICENSE=accept <image name>` will print a listing of all the command with a short description.
-For detailed help on any command, run `docker run -v`pwd`:/share -e CHEF_LICENSE=accept <image name> help [COMMAND]`. Help can also be called with the `-h, --help` flags after any command, like `docker run -v`pwd`:/share -e CHEF_LICENSE=accept <image name> xccdf2inspec -h`.
+Building the Docker container and running it with `docker run -v` `` ` ``pwd`` ` `` `:/share -e CHEF_LICENSE=accept <image name>` will print a listing of all the command with a short description.
+For detailed help on any command, run `docker run -v` `` ` ``pwd`` ` `` `:/share -e CHEF_LICENSE=accept <image name> help [COMMAND]`. Help can also be called with the `-h, --help` flags after any command, like `docker run -v` `` ` ``pwd`` ` `` `:/share -e CHEF_LICENSE=accept <image name> xccdf2inspec -h`.
```

Comment:
This is not rendering correctly, you will need to put \`\`\` on each side of your docker run command for it to display correctly.